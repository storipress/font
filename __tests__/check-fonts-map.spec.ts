import { expect, it } from 'vitest'

it('expect fonts map unchanged', async () => {
  let map
  try {
    map = await import('../font-list')
  } catch {
    expect.fail('font-list.ts not found')
  }
  expect(map).toMatchSnapshot()
})
