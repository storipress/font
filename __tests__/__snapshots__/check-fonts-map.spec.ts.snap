// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`expect fonts map unchanged 1`] = `
{
  "fontList": {
    "Condensed": [
      "Fibel Nord",
      "IBM Plex Sans Condensed",
      "Junicode Condensed",
      "League Gothic",
      "Tex Gyre Heros Condensed",
      "Tuffy",
    ],
    "Display": [
      "Aileron Black",
      "Aileron Light",
      "Anderson Grotesk Black",
      "Archivo Black",
      "Avara Bold",
      "BNBoop",
      "BNBrick-House",
      "BNChester",
      "BNGrimer",
      "BNReconGrotesk",
      "BNRiptide",
      "Backout",
      "Bagnard",
      "Bagnard Sans",
      "Bebas Neue",
      "Bizmeud",
      "Blocus",
      "Bluu Next Bold",
      "Butler",
      "Butler Stencil",
      "Canola Sans Compressed",
      "Canola Slab Compressed Medium",
      "Chicago FLF",
      "Combat",
      "Commune Nuit Debout",
      "Commune Nuit Debout Pochoir",
      "Cooper Hewitt Heavy",
      "DOF 3 Extended",
      "Fengardo Neue",
      "Fibel Nord Kontur",
      "Fira Sans Black",
      "Fivo Sans Modern Heavy",
      "Format 1452",
      "Gap Sans",
      "Grotesk",
      "Gulax",
      "Half",
      "Happy Times at the IKOB",
      "Jost Hevy",
      "Karrik",
      "Lack",
      "Lack Line",
      "Le Murmure",
      "Libertinus Serif Display",
      "Metropolis Black",
      "Millimetre",
      "Minipax",
      "Mister Pixel",
      "Mono",
      "Monoton",
      "Objectivity Super",
      "Oswald",
      "Quebec",
      "Redaction",
      "Redaction 70 Bold",
      "Savate",
      "Shapiro Air Extd",
      "Shapiro Feather Text",
      "Shapiro Light",
      "Soyuz Grotesk Bold",
      "Sporting Grotesque",
      "Sprat",
      "Sprat Condensed Bold",
      "Sprat Extended",
      "Subjectivity Serif Super",
      "Syne",
      "Syne Extra",
      "Syne Mono",
      "Terminal Grotesque",
      "Terminal Grotesque Open",
      "Toren Rotalic",
      "VG5000",
      "Victorianna Thin",
      "W95FA",
      "Wremena",
      "Wremena Light",
      "Xolonium",
      "Young Serif",
    ],
    "Mono": [
      "AnkaCoder",
      "Anonymous Pro.001",
      "JetBrains Mono.001",
      "Office Code Pro",
      "Roboto Mono",
      "Sometype Mono",
      "Space Mono",
      "Ubuntu Mono",
      "Victor Mono",
      "iA Writer Quattro",
      "luculent",
    ],
    "Sans": [
      "Aileron",
      "Anderson Grotesk",
      "Anodina",
      "Archivo",
      "Asap",
      "Bai Jamjuree",
      "Barlow",
      "Chivo",
      "Cooper Hewitt",
      "DejaVu Sans",
      "Fira Sans",
      "Fivo Sans Modern",
      "HK Grotesk",
      "IBM Plex Sans",
      "Ilisarniq",
      "Inria Sans",
      "Inter",
      "Jost",
      "Kanit",
      "Lato",
      "Liberation Sans",
      "Libre Franklin",
      "Luiss Sans",
      "Metropolis",
      "Montserrat",
      "Nimbus Sans",
      "Noto Sans Malayalam",
      "Nunito",
      "Objectivity",
      "Open Sans",
      "Open Sauce Sans",
      "Oswald",
      "Overpass",
      "PT Sans",
      "Poppins",
      "Prompt",
      "Proza Libre",
      "Public Sans",
      "Red Hat Display",
      "Red Hat Text",
      "Reforma 2018",
      "Roboto",
      "Rubik",
      "Secuela",
      "Sinkin Sans",
      "Source Sans Pro",
      "Standard",
      "Subjectivity Serif",
      "Tex Gyre Heros",
      "Wonder Unit Sans",
      "Work Sans",
      "iA Writer Duo",
    ],
    "Serif": [
      "Amiri",
      "Bitter Pro",
      "Caladea",
      "Charis SIL",
      "Cormorant Garamond",
      "Crimson Pro",
      "EB Garamond",
      "Gelasio",
      "IBM Plex Serif",
      "Ibarra Real Nova",
      "Inria Serif",
      "Junicode",
      "Libertinus Serif",
      "Lora",
      "Luis Serif",
      "Manuale",
      "Noto Serif",
      "PT Serif",
      "Playfair Display",
      "Reforma 1918",
      "Reforma 1969",
      "Tinos",
      "Vollkorn",
    ],
  },
}
`;
